spec/spec_helper.rb

```diff
@@ -45,6 +91,22 @@
     mocks.verify_partial_doubles = true
   end
 
+  config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+    start_es_server unless ENV['ES_SKIP']
```

Comment:
my workflow is to run specs automatically from within the code using a vim-rspec plugin. Yoz does something similar with atom. running the above, while possible, would not work with our current rspec workflows. 

the larger concern I have is that running a single spec where you *do* want to start the es server would not work unless that spec file is run within the suite. does that make sense? perhaps I am wrong but I would expect that specs requiring the es server would only work when run within the test suite, which would present a problem for all spec running workflows.